lib/vector-lib/Cargo.toml

```diff
@@ -6,7 +6,10 @@ edition = "2021"
 publish = false
 
 [dependencies]
-vector-common = { path = "../vector-common", default-features = false }
+vector-common = { path = "../vector-common" }
+vector-config = { path = "../vector-config" }
+vector-config-common = { path = "../vector-config-common" }
+vector-config-macros = { path = "../vector-config-macros" }
```

Comment:
I wonder if, actually, with your changes.... needing to import `vector-config-common` and `vector-config-macros` directly can now be dropped? 🤔

Reply:
If so, it would also let you significantly shrink/clean up the `pub mod configurable` block in `vector-lib`.

Reply:
Great catch! Fixed in 7983942e1